README.md

```diff
@@ -5,13 +5,13 @@ The project composes of the mobile version of my personal portfolio. The project
 
 # Live
 
-https://b77748.github.io/arnaud_portfolio/
+https://b77748.github.io/arnaud_portfolio.github.io/
 
 # Built With
 
-•	HTML
-•	CSS
-•	Linters
+-	HTML
+-	CSS
+-	Linters
 
 # Getting Started
```

Comment:
- [ ] In README FIle, the `Getting started ` section should contain the necessary code to run the project locally.
```
git clone https://github.com/B77748/arnaud_portfolio.git
cd arnaud_portfolio
```
Right-click on your opened index.html file and launch the live server to preview